strapi/strapi
# fix: Crash in the admin panel when i18n default locale is unset

### What does it do?

Prevent the issue with the app crashing  when we attempt to `add/edit/delete` Collection Types `entry` in `Content Manager` when the default locale is unset

### Why is it needed?
App crashes when we attempt to `add/edit/delete` Collection Types `entry` in `Content Manager` when the default locale is unset

### How to test it?

1. Create and launch strapi app
2. Remove the plugin_i18n_default_locale row in the strapi_core_store_settings table in your database
3. Go to the content manager and try to edit or add any content type
4. Verify that there are no errors, and everything still works fine.
 
Demos

https://user-images.githubusercontent.com/7220167/205881503-66d9ab0e-ecf1-4e90-a1a3-8137a002ef22.mov


### Related issue(s)/PR(s)

- Fixes https://github.com/strapi/strapi/issues/14313
